client/app/cloud/project/translations/Messages_fr_FR.json

```diff
@@ -2,5 +2,10 @@
   "cpc_tab_billing": "Consommation",
   "cpc_tab_openstack_users": "Openstack",
   "cpb_details_compute_error_message": "Oups, nous n'arrivons pas à récupérer les informations concernant les infrastructures.",
-  "cpb_read_only_access_alert": "Votre compte est en lecture seule, certaines opérations sont inaccessibles. Pour obtenir un privilège d'écriture, contactez votre administrateur."
+  "cpb_read_only_access_alert": "Votre compte est en lecture seule, certaines opérations sont inaccessibles. Pour obtenir un privilège d'écriture, contactez votre administrateur.",
+  "cpc_header_tab_compute": "Compute",
+  "cpc_header_tab_tech_mgmt": "Gestion technique",
+  "cpb_header_tab_admin_mgmt": "Gestion administrative",
+  "cpc_header_tab_localisation": "Ajout de localisation",
+  "cpc_header_tab_openstack": "OpenStack users"
 }
```

Comment:
I know you didn't change this, but maybe you can fix this missing new line ? (Mayeb you should check your editor if it has an editorconfig with the following setting `insert_final_newline = true`)